src/frontends/onnx/frontend/src/op/com.microsoft/fused_conv.cpp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "op/com.microsoft/fused_conv.hpp"
+
+#include <memory>
+#include <vector>
+
+#include "default_opset.hpp"
+#include "exceptions.hpp"
+#include "op/conv.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_1 {
+OutputVector fused_conv(const Node& node) {
+    const auto conv_res = conv(node).at(0);
```

Comment:
I'm just wondering, what is the purpose of the optional input `Z` mentioned in the documentation.
As I can see it's not used during import.

Reply:
Great catch! Thank you, `Z` input is just added to conv result before actication (if provided). Added